lib/teleterm/gateway/kube_cert_reissuer.go

```diff
@@ -37,7 +37,8 @@ func newKubeCertReissuer(cert tls.Certificate, onExpiredCert func(context.Contex
 	r := &kubeCertReissuer{
 		onExpiredCert: onExpiredCert,
 	}
-	r.updateCert(cert)
+	// Ignoring the error since we know that updateCert always returns nil.
+	_ = r.updateCert(cert)
```

Comment:
At first I rewrote `newKubeCertReissuer` so that it returns an error and checks for the error from `updateCert`. However, when reviewing the PR, I noticed why `updateCert` returns an error in the first place. So instead I just explicitly ignored the error.

idk if that's okay, certainly better than "implicitly" ignoring it but maybe it's better to actually check it even if we know that it always returns nil? In one way it helps with future refactoring because we make sure that if `updateCert` starts throwing errors, they will be checked. On the other hand, it makes future refactoring more complicated because people touching `newKubeCertReissuer` might not realize that that function does not need to return an error.

Reply:
Definitely bad writing when I first implemented it. I don't mind the current approach. As an alternative, what do you think about dropping the error on the `updateCert` functions, and wrap this function when used as a callback, something like
```go
    k.onNewCertFuncs = append(k.onNewCertFuncs, func(cert tls.Certificate) error {
        certReissuer.updateCert(cert)
        return nil
    })
```

Reply:
Oh, if you need to return an error in the future,  i think it makes that `newKubeCertReissuer` returns an error as well.

Reply:
I probably won't need the error, I was just describing a hypothetical situation.

I thought of the idea of wrapping `updateCert` as well but I disregarded it at first because ` k.onNewCertFuncs = append(k.onNewCertFuncs, certReissuer.updateCert)` looked better and I felt that I'm not introducing new abstractions unnecessarily. But that wrapping function is hardly an abstraction at all, so I think I'm just going to change the implementation to this!